src/test/utility/processor.test.js

```diff
@@ -16,13 +17,13 @@ const OrgUnit = {
 const TestModule = {
 	title: 'Test Module',
 	type: 'module',
-	descriptionFileName: 'test-module/index.md',
+	descriptionFileName: 'test-module/index.html',
```

Comment:
I think these should remain `.md` as they're simulating the original manifest, otherwise `fileHandler` will not find the file.

Reply:
Oh, I see this is using the pre-converted test files... My mistake.

Reply:
Yea, I wanted to isolate the markdown rendering to the filehandler.  Other tests/modules shouldnt have to care.